### PR TITLE
Mv scale up

### PR DESCRIPTION
**Pull request by huwshimi**
Add service scaling from the machine view.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- use `/:flags:/mv/`
- add some services
- open the machine view
- in the header of the unplaced units column click the +
- enter some numbers of units for the services
- click Add units
- units should be added for the services

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3921/
Test PASSed.

File: jujugui/static/gui/src/app/components/generic-button/generic-button.js [hunk 1]
```diff
@@ -21,6 +21,10 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 YUI.add('generic-button', function() {
 
   juju.components.GenericButton = React.createClass({
+    propTypes: {
```
Comment (kadams54) [inline jujugui/static/gui/src/app/components/generic-button/generic-button.js]: Yay! I'd like to see propTypes popping up in more existing React components.

**Comment by kadams54** [comment]
QA is :ok: and :+1: on the code. Thanks!

QA Notes:

Added the mongodb cluster bundle. Switched to machine view and clicked on the plus widget. Added varying numbers of units to each service and clicked on "Add units." Verified that the units were added as expected. Repeated and verified that the cancel button works. Tried to add -1 and "foo" units.

**Comment by huwshimi** [comment]
Thanks @kadams54! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1345

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**